.github/workflows/snyk.yml

```diff
@@ -0,0 +1,25 @@
+# ref: https://github.com/snyk/actions/tree/master/gradle-jdk11
+name: snyk for gradle
+on:
+  push:
+  release:
```

Comment:
Why run on release? Nothing would have changed from the target commit right?

Reply:
same. will arrange triggers while converting to PR. Ronak was suggesting some rearrangements in workflows and jobs will add those and change triggers.